Nighttime-Operations/Observatory/DIMM/index.rst

```diff
@@ -4,11 +4,52 @@
 DIMM
 ###############
 
-This page contains some common calibrations procedures.
+The differential image motion monitor (DIMM) is a standard instrument to measure astronomical "seeing", or atmospheric turbulence strength. 
+
+The DIMM principle is to produce "twin images" of a bright star by a mask with two apertures and a wedge prism at the entrance pupil of the telescope. 
+The rms of the fluctuations of the angular distance between the two images is proportional to the atmospheric optical turbulence, that in turn, allows to predict the seeing FWHM. 
+
+The main control software is divided into several programs. The main services are `ameba` that is in charge of the observation, synchronizing dome, weather service, 
+mount and camera control, and the `preat` program that converts the output of the `dimm_tool` into seeing values.
+
+.. figure:: ./_static/program_structure.png
+    :name: DIMM program structure
+
+    Structure of the DIMM software
+
+
+Below is a list of some of the most useful DIMM manuals:
```

Comment:
Is there also a collection on docushare where all the DIMM documents are? If so, then it'd be useful to link that at the bottom... just so people know where to hunt for more stuff if needed.

Reply:
Yes, working on that with David Cabrera.